js/src/classic-editor.js

```diff
@@ -121,10 +121,11 @@ jQuery(
 					dialogResult = Promise.resolve();
 				}
 
-				// phpcs:disable PEAR.Functions.FunctionCallSignature.EmptyLine
 				dialogResult.then(
 					() => {
-						var data = {  // phpcs:ignore PEAR.Functions.FunctionCallSignature.Indent
+						var lang  = selectedOption.options[selectedOption.options.selectedIndex].lang;
+						var dir   = $( '.pll-translation-column > span[lang="' + lang + '"]' ).attr( 'dir' );
```

Comment:
These variables are no longer defined normally, so there might have been a problem with the rebase.

Reply:
🤔 Strange! Because I rebased from origin/master and it didn't detect conflict there. I'm looking at that.

Reply:
Hum! I see my mistake. I totally make confidence to the comparison. Thanks.